test/fixtures/rules/std_rule_cases/L007.yml

```diff
@@ -98,20 +136,85 @@ passes_on_after_override:
 fails_on_before_override_with_comment_order:
   fail_str: |
     select
-        a
```

Comment:
Why did we change an existing test rather than adding a new one?

Reply:
It seemed better to extend the existing test (see the addition of a third comment), which make this test cover additional functionality, than just adding in another test and having two tests which test almost the same thing.